{"name": "testData[CaseData4]", "status": "broken", "statusDetails": {"message": "KeyError: 'casename'", "trace": "self = <testcase.test_excel.TestCase object at 0x0000018894D46530>\nCaseData = {'actualResult': '$..msg', 'caseName': '请求方法错误', 'datas': '{\"accounts\": \"hami\",\"pwd\": \"123456\",\"type\": \"username\"}', 'expectResult': '登录成功', ...}\n\n    @pytest.mark.parametrize(\"CaseData\",AllCaseData)\n    def testData(self,CaseData):\n    \n        # 调用动态生成标题\n        # self.__dynmic_title(CaseData)\n>       casename=\"ID({})-({})\".format(CaseData[\"id\"],CaseData[\"casename\"])\nE       KeyError: 'casename'\n\ntest_excel.py:42: KeyError"}, "parameters": [{"name": "CaseData", "value": "{'id': 7, 'caseName': '请求方法错误', 'url': 'http://shop-xo.hctestedu.com/index.php?s=/api/user/login', 'method': 'get', 'headers': '{\"Content-Type\":\"application/json\"}', 'datas': '{\"accounts\": \"hami\",\"pwd\": \"123456\",\"type\": \"username\"}', 'actualResult': '$..msg', 'expectResult': '登录成功', 'result': '数据提取失败，请检查参数', 'response': '日志为空，数据提取失败，请检查参数'}"}], "start": 1727245006769, "stop": 1727245006769, "uuid": "d32768e2-aed0-482e-b7d2-c22a23081c31", "historyId": "78e81b98d714df2ff221541a11b8a611", "testCaseId": "92e66718ddbc07993fbf10087035b16e", "fullName": "test_excel.TestCase#testData", "labels": [{"name": "suite", "value": "test_excel"}, {"name": "subSuite", "value": "TestCase"}, {"name": "host", "value": "DESKTOP-EA3KTC1"}, {"name": "thread", "value": "18592-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_excel"}]}